themes/survey/fruity/views/subviews/header/custom_header.twig

```diff
@@ -139,7 +139,7 @@
         .question-container {
             animation-name: {{ aSurveyInfo.options.questionanimation  }};
             animation-duration: {{ questionanimationduration }}ms;
-            animation-fill-mode: both;
+            animation-fill-mode: none;
```

Comment:
Since non is default : maybe remove ?

Reply:
I would just leave it like that as to make sure this will not be inherited and also to reflect we want this to be default explicitely

Reply:
Maybe add the mantis id in twig comment ?